google/resource_container_node_pool.go

```diff
@@ -59,7 +50,12 @@ func resourceContainerNodePool() *schema.Resource {
 				},
 				"cluster": &schema.Schema{
 					Type:     schema.TypeString,
-					Required: true,
+					Optional: true,
```

Comment:
Why is `Cluster` optional now?

Reply:
I can revert this back, my original design was to expose the `parent` parameter and expect the TF user to construct the string to match. As we have gone with `region` this can definitely be reverted.